src/test/regress/expected/single_shard_table_udfs.out

```diff
@@ -0,0 +1,219 @@
+CREATE SCHEMA null_dist_key_udfs;
+SET search_path TO null_dist_key_udfs;
+SET citus.next_shard_id TO 1720000;
```

Comment:
this is the same shardid used in create_single_shard_table.sql?

Reply:
yes, is it a problem for us?